Installations instructions corrected

AryanNanda17 opened the pull request:
This PR resolves issue number #665
I have reviewed and updated the installation instructions mentioned [here](https://jderobot.github.io/BehaviorMetrics/install).

I have done these changes:-
 - Added instructions to download Ros1 noetic dependencies. 
 - Changed make build to sudo make build (else Permission denied error was coming).
 - Added instructions to download catkin_pkg else getting this error:- 
<img width="668" alt="Screenshot 2024-03-03 at 2 21 21 PM" src="https://github.com/JdeRobot/BehaviorMetrics/assets/125150482/22828823-a5fd-4837-8842-6c1892e3d2eb">

- Specified to install python3 version 3.8 or below because the dependencies of Behavior-metrics is compatible with it only else getting the following error. 
<img width="908" alt="Screenshot 2024-03-03 at 2 26 32 PM" src="https://github.com/JdeRobot/BehaviorMetrics/assets/125150482/49ba7b5c-2c56-4958-a74c-ac9e26e7093f">
<img width="774" alt="Screenshot 2024-03-03 at 2 29 01 PM" src="https://github.com/JdeRobot/BehaviorMetrics/assets/125150482/22877e70-3844-4710-91b7-117112a0790a">

AryanNanda17 committed:
Installations instructions corrected

AryanNanda17 commented:
Moreover, @sergiopaniego the installation guide for Carla directs [here](https://carla.readthedocs.io/en/0.9.13/start_quickstart/) but, I was getting this error:-
<img width="722" alt="Screenshot 2024-03-03 at 2 31 20 PM" src="https://github.com/JdeRobot/BehaviorMetrics/assets/125150482/6f154098-90cd-4e10-a09d-bb77a66f347d">

I researched and I found that the server seems to be down for quite some time. I would suggest adding the manual instructions to the main docs. 
[This](https://github.com/carla-simulator/carla/issues/7017) issue describes the problem.

sergiopaniego commented:
Thank you for opening the PR.
The CustomRobots repository is only needed for launching Behavior Metrics with Gazebo. Since our main simulator is CARLA, we should include this information on the web. Could you check running the application without CustomRobots and send a video of Behavior Metrics running?

sergiopaniego commented:
Regarding the issue with the installation of the CARLA package, we are aware of this issue (https://github.com/JdeRobot/BehaviorMetrics/issues/657). The server has been down for a few weeks, so I think we should include instructions for the manual installation too. Please, add them!

sergiopaniego commented:
Please @AryanNanda17, review the PR since we have updated the installation instructions to 0.9.15 in this PR: https://github.com/JdeRobot/BehaviorMetrics/pull/670.
The changes should not affect this development
Thank you so much for your contribution! :smile:

AryanNanda17 committed:
Manual instructions of CARLA added

AryanNanda17 committed:
Merge conflict removed

AryanNanda17 committed:
CustomRobot instructions added

AryanNanda17 commented:
Hello @sergiopaniego, 
Apologies for the delay, but I'm currently in the midst of my mid-semester exams.


> Regarding the issue with the installation of the CARLA package, we are aware of this issue (#657). The server has been down for a few weeks, so I think we should include instructions for the manual installation too. Please, add them!

I have added instructions for Carla's manual installations. 

> Robots repository is only needed for launching Behavior Metrics with Gazebo. Since our main simulator is CARLA,

I have added the required instructions.

AryanNanda17 committed:
Typo corrected

AryanNanda17 commented:
Hello @sergiopaniego,
Please review the PR and suggest if any changes are required.

sergiopaniego commented:
Thanks for the contribution! Merged :smile:
